backend/core/src/listings/listings.controller.ts

```diff
@@ -57,6 +58,26 @@ export class ListingsController {
     return mapTo(PaginatedListingDto, await this.listingsService.list(queryParams))
   }
 
+  @Get("includeExternal")
+  @ApiExtraModels(ListingFilterParams, ListingsQueryParams)
+  @ApiOperation({
+    summary: "List listings and optionally include external listings",
+    operationId: "listIncludeExternal",
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @UsePipes(new ValidationPipe(defaultValidationPipeOptions))
+  public async getAllWithExternal(
+    @Query() queryParams: DoorwayListingsExternalQueryParams
+  ): Promise<ListingIncludeExternalResponse> {
+    const jurisdictions: string[] = queryParams.bloomJurisdiction
+    // See doorway-listings-external-query-params.ts for more context on why
+    // we delete this part of the params and typecast.
+    delete queryParams.bloomJurisdiction
```

Comment:
is it practical to use a decorator that does this? if so, we could re-used this pattern in other methods.

Reply:
I am very not well versed in typescript/js 😅 could you elaborate and/or send me a link describing the design pattern you're talking about?

Reply:
Please disregard my note about decorators. Doesn't `mapTo` remove unused properties for us if we pass in `{ excludeExtraneousValues: true }`

https://github.com/metrotranscom/doorway/blob/5c0d991dbaecf90924c90d16dee4e366c7e13bf5/backend/core/src/shared/mapTo.ts#L5

https://github.com/typestack/class-transformer#enforcing-type-safe-instance

Reply:
Given how unique this particular bit of functionality is, does it need to be made into a repeatable pattern?

Reply:
If it is unique, then no. 

I think we might be able to remove L75 tho

Reply:
That's a good point.  With `excludeExtraneousValues` set to true it should prevent `bloomJurisdiction` from being added to the ListingsQueryParams object automatically, so it's safe to discard the explicit delete

Reply:
FWIW weirdly this does not seem to actually `excludeExtraneousValues` however I think that /listings is set up to ignore the param so this is fine as-is.

https://screenshot.googleplex.com/9SAeosu3yrhoCBW